frontend/pages/hosts/details/cards/Software/SelfService/CategoriesMenu/CategoriesMenu.tsx

```
@@ -21,35 +23,37 @@ const CategoriesMenu = ({
-              ...queryParams,
-              page: 0,
-              category_id: cat.id !== 0 ? cat.id : undefined,
-            }}
-            to={location.pathname}
-            className={classNames({
-              [`${baseClass}__category-link`]: true,
+          <li
+            className={classNames(`${baseClass}__category-link`, {
               [`${baseClass}__category-link--active`]: isActive,
             })}
           >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)